great_expectations/render/types/__init__.py

```diff
@@ -501,44 +508,91 @@ class RenderedAtomicValue(DictDot):
     def __init__(
         self,
         schema: Optional[dict] = None,
+        header: Optional["RenderedAtomicValue"] = None,
```

Comment:
Should we add `# noqa` here?

Reply:
It looks like we're able to annotate properly on L520. Could we do that throughout the file if we don't get cyclic import issues?

Reply:
Something about the fact that this class has self-referential type hints means that you need to quote in the signature.

Reply:
I get "Unresolved reference" when the quotes are removed.

Reply:
ah okay good call, thank you!